player/src/androidTest/kotlin/com/tidal/sdk/player/playlog/PlayLogTest.kt

```diff
@@ -176,8 +177,8 @@ class PlayLogTest {
             eq(ConsentCategory.NECESSARY),
             argThat {
                 with(Gson().fromJson(this, JsonObject::class.java)["payload"].asJsonObject) {
-                    assertThat(get("startAssetPosition").asDouble).isEqualTo(0.0)
-                    assertThat(get("endAssetPosition").asDouble).isBetween(5.0, 5.1)
+                    assertThat(get("startAssetPosition").asDouble).isAssetPositionEqualTo(0.0)
```

Comment:
Should we link to [this](https://github.com/androidx/media/issues/1252) and [this](https://github.com/androidx/media/issues/1253) so we can correct it if they ever get fixed?

Reply:
done